.vscode/tasks.json

```diff
@@ -0,0 +1,41 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "dist",
+			"problemMatcher": [],
+			"label": "Test",
+			"detail": "yarn test",
+			"group": {
+				"kind": "test",
+				"isDefault": true
+			}
+		},
+		{
+			"type": "npm",
+			"script": "dist",
+			"problemMatcher": [],
+			"label": "Build for Linux",
+			"detail": "yarn build-electron && yarn build && electron-builder -c.extraMetadata.main=build/main.js --linux",
```

Comment:
We are missing the targeting here. I'm not sure if the default is appImage, but if not, would be good to pass it after the `--linux`

Reply:
I think that it is a default since i generated the task from package.json. It'd be good to test it tho.

Reply:
confirmed that it generates an appimage